### PR TITLE
created word2vec downloader

### PR DESCRIPTION
**Pull request by MartinXPN**
Issue #8 
Created word2vec downloader to download glove zip file extract it and convert it to .vec format 
Also updated README file to indicate that one first needs to download word2vec data and then proceed with next steps

### CODE REVIEW

File: preprocessing.py [hunk 1]
```diff
@@ -35,7 +36,16 @@ def tokenize_context(context):
 
 
 def word2vec(word2vec_path):
+    # Download word2vec data if it's not present yet
+    if not path.exists(word2vec_path):
```
Comment (mahnerak) [inline preprocessing.py]: Utility function `get_file` is able to check whether the requested file exists or not.

Comment (MartinXPN) [reply preprocessing.py]: this has nothing to do with get_file, it's the output of glove2word2vec convertion

Comment (MartinXPN) [reply preprocessing.py]: the function name get_word2vec_zip_path might be confusing and incorrect as it returns the file path of unzipped glove file so I updated it in the last commit

**Comment by mahnerak** [comment]
LGTM

**Pull request merged**